third_party/llvm/setup.bzl

```diff
@@ -3,7 +3,7 @@
 load("@llvm-raw//utils/bazel:configure.bzl", "llvm_configure", "llvm_disable_optional_support_deps")
 
 # The subset of LLVM targets that TensorFlow cares about.
-_LLVM_TARGETS = [
+supported_targets = [
```

Comment:
It would significantly simply internal adjustments if you could rename this to "llvm_targets".